packages/tracking/src/Tracking.php

```diff
@@ -1,30 +1,25 @@
 <?php
 /**
  * Nosara Tracks for Jetpack
+ *
+ * @package jetpack-tracking
  */
 
 namespace Automattic\Jetpack;
 
-require_once dirname( dirname( dirname( __DIR__ ) ) ) . '/_inc/lib/tracks/client.php';
-
+/**
+ * The Tracking class, used to record events in wpcom
+ */
 class Tracking {
-	static $product_name = 'jetpack';
-
-	static function track_jetpack_usage() {
-		if ( ! \Jetpack::jetpack_tos_agreed() ) {
-			return;
-		}
-
-		// For tracking stuff via js/ajax
-		add_action( 'admin_enqueue_scripts', array( __CLASS__, 'enqueue_tracks_scripts' ) );
+	private $product_name;
+	private $connection;
 
-		add_action( 'jetpack_activate_module', array( __CLASS__, 'track_activate_module' ), 1, 1 );
-		add_action( 'jetpack_deactivate_module', array( __CLASS__, 'track_deactivate_module' ), 1, 1 );
-		add_action( 'jetpack_user_authorized', array( __CLASS__, 'track_user_linked' ) );
-		add_action( 'wp_login_failed', array( __CLASS__, 'track_failed_login_attempts' ) );
+	function __construct( $product_name = 'jetpack' ) {
+		$this->product_name = $product_name;
+		$this->connection   = new Connection\Manager();
```

Comment:
Maybe have a `use Automattic\Jetpack\Connection\Manager as Connection_Manager` on top to make this more specific? Because if we end up moving this line of code, it'll be prone to errors.

Reply:
we should kill the Manager and make it a simple Connection at some point

Reply:
Agreed

Reply:
Leaving as-is for now.